vpd-tool/src/vpd_tool_main.cpp

```diff
@@ -38,13 +49,27 @@ int main(int argc, char** argv)
     auto l_fileOption = l_app.add_option("--file", l_filePath,
                                          "Absolute file path");
 
+    auto l_keywordValueOption =
+        l_app.add_option("--value, -V", l_keywordValue,
+                         "Keyword value in ascii/hex format."
+                         " ascii ex: 01234; hex ex: 0x30313233");
+
+    auto l_hardwareFlag = l_app.add_flag("--Hardware, -H",
+                                         "CAUTION: Developer only option.");
+
     auto l_readFlag = l_app.add_flag("--readKeyword, -r", "Read keyword")
                           ->needs(l_objectOption)
                           ->needs(l_recordOption)
                           ->needs(l_keywordOption);
 
-    auto l_hardwareFlag = l_app.add_flag("--Hardware, -H",
-                                         "CAUTION: Developer only option.");
+    auto l_writeFlag =
+        l_app
+            .add_flag(
+                "--writeKeyword, -w",
+                "Write keyword, Note: Irrespective of dbus or hardware path provided, both DBus and hardware paths will get updated with given key value")
```

Comment:
> Write keyword, Note: Irrespective of dbus or hardware path provided, both DBus and hardware paths will get updated with given key value

 Do we need to mention that Primary and Backup paths will get updated along with Redundant EEPROM(if any)?